state_publisher_rate: 150.0
velocity_publisher_rate: 10.0
simulation: True

crazyflies:
  - name: cf_1
    active: True
    uri: radio://0/80/2M/E7E7E7E7E4
    sim_uri: udp://0.0.0.0:19850
    takeoff_height: 0.2
    takeoff_duration: 3
    multiranger: True
    initial_position:
      x: 0.0
      y: 0.0
      z: 0.0

  - name: cf_2
    active: False
    uri: radio://0/80/2M/E7E7E7E7E5
    sim_uri: udp://0.0.0.0:19851
    takeoff_height: 0.2
    takeoff_duration: 3
    multiranger: True
    initial_position:
      x: 1.0
      y: 0.0
      z: 0.0

  - name: cf_3
    active: False
    uri: radio://0/80/2M/E7E7E7E7E6
    sim_uri: udp://0.0.0.0:19852
    takeoff_height: 0.2
    takeoff_duration: 3
    multiranger: False
    initial_position:
      x: 0.0
      y: 0.0
      z: 0.0

  - name: cf_4
    active: False
    uri: radio://0/80/2M/E7E7E7E7E7
    sim_uri: udp://0.0.0.0:19853
    takeoff_height: 0.3
    takeoff_duration: 3
    multiranger: True
    initial_position:
      x: 0.0
      y: 0.0
      z: 0.0

  - name: cf_5
    active: False
    uri: radio://0/80/2M/E7E7E7E7E8
    sim_uri: udp://0.0.0.0:19854
    takeoff_height: 0.3
    takeoff_duration: 3
    multiranger: True
    initial_position:
      x: 0.0
      y: -1.0
      z: 0.0

  - name: cf_6
    active: False
    uri: radio://0/80/2M/E7E7E7E7E9
    sim_uri: udp://0.0.0.0:19855
    takeoff_height: 0.5
    takeoff_duration: 3
    multiranger: True
    initial_position:
      x: 0.5
      y: -0.5
      z: 0.0
